blockbook.go

```diff
@@ -196,6 +196,17 @@ func mainWithExitCode() int {
 		}
 		internalState.UtxoChecked = true
 	}
+
+	// sort addressContracts if necessary
+	if !internalState.SortedAddressContracts {
+		err = index.SortAddressContracts(chanOsSignal)
+		if err != nil {
+			glog.Error("sortAddressContracts: ", err)
+			return exitCodeFatal
+		}
+		internalState.SortedAddressContracts = true
```

Comment:
you had a comment about sync here that I wasn't exactly sure what you meant if you want to provide additional detail

Reply:
I did not want to set the migration as complete until the sorting is implemented in the `addToContract` function. If I just sort and then run the old version, new data will be inserted in wrong order...

Reply:
Oh right, makes sense thanks.